studiopress/pattern-manager
# Fix some issues with block pattern previews

## Tasks


### Summary of changes
This PR fixes some issues with block pattern previews, especially ones relating to Genesis Framework.

### How to test
1. Install a Genesis theme, like Navigation Pro, and save the home page blocks to a pattern using Pattern Manager
2. Notice the pattern preview is not accurate to how the pattern should look. 
3. On this branch, it should look correct again

### Notes & Screenshots

#### Before this branch (broken)
<img width="1284" alt="Screenshot 2023-05-01 at 4 05 32 PM" src="https://user-images.githubusercontent.com/7538525/235521874-9b288d3c-f2b7-4297-b336-93a24791b5ea.png">

### On this branched (fixed)
<img width="1358" alt="Screenshot 2023-05-01 at 4 05 54 PM" src="https://user-images.githubusercontent.com/7538525/235521997-4acfbff4-0adb-45cd-b4c5-628b9fe07f8f.png">
